linkml/generators/docgen.py

```diff
@@ -959,6 +965,12 @@ def example_object_blobs(self, class_name: str) -> list[tuple[str, str]]:
                 objs.append((stem, f.read()))
         return objs
 
+    def customize_environment(self, env: Environment):
+        if self.truncate_descriptions:
+            env.filters["enshorten"] = enshorten
+        else:
+            env.filters["enshorten"] = text_to_web
```

Comment:
rather than edit all the templates, this just switches out the `enshorten` function, which truncates text, for a function that converts line breaks to `<br>` elements for web display (any `\n` line breaks in description elements break the layout of markdown tables).